crates/vault-registry/src/lib.rs

```diff
@@ -849,6 +849,12 @@ impl<T: Config> Pallet<T> {
             Err(x) => return Err(x),
         };
 
+        ensure!(
+            new_collateral.is_zero()
+                || new_collateral.ge(&Self::get_minimum_collateral_vault(vault_id.currencies.collateral))?,
```

Comment:
This allows the vault operator to withdraw everything if there is a nominator with sufficient collateral. Afterwards, the nominator may not be able to withdraw. Maybe we should do the check only for the vault operator?

Reply:
Agree that should probably be the case but that is a separate feature since that behavior is anyway possible without my change and nomination is still disabled.

Reply:
Ok, made an issue